### PR TITLE
Improve menu navigation and add discord button

### LINKED ISSUE #464: Add support/Discord button

### LINKED ISSUE #471: Cleanup app bar
plus add discord help
report bug - link to github issues

### LINKED ISSUE #472: Add alert on reload
"Reloading the page will require to start all the validators again. Are you sure?"

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #464 
closes #471 
closes #472

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -4,3 +4,6 @@ SENTRY_ORG=nodefactory
 SENTRY_PROJECT=chainguardian
 SENTRY_AUTH_TOKEN=xxx
 DOCKER_LIGHTHOUSE_IMAGE=sigp/lighthouse:v1.1.0
+DISCORD_URL=https://discord.gg/4GBwH52cFb
+CHAINGUARDIAN_URL=https://chainguardian.nodefactory.io/
+GITHUB_URL=https://github.com/NodeFactoryIo/ChainGuardian
```
Comment (mpetrunic) [inline .env.dist:9]: I think you can hardcode those  in js. It doesn't change between environments

**Review by mpetrunic — CHANGES_REQUESTED** [review]

File: .env.dist (hunk 1 above)
Comment (BeroBurny) [reply .env.dist:9]: agree only on Github URL, but for discord and app URL is good to have in ENV in case of change

Comment (mpetrunic) [reply .env.dist]: But even if it change, we would have to make new release anyways?

Comment (BeroBurny) [reply .env.dist]: yes

File: src/main/menu/menu.ts [hunk 2]
```diff
@@ -59,19 +59,19 @@ const template = [
             {
                 label: "Discord Support",
                 click: async (): Promise<void> => {
-                    await shell.openExternal(process.env.DISCORD_URL);
+                    await shell.openExternal("https://discord.gg/4GBwH52cFb");
```
Comment (mpetrunic) [inline src/main/menu/menu.ts:62]: Ain't this link suppose to be different like inviting to specific support channel?

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**